keyboards/dichotomy/rules.mk

```diff
@@ -1,6 +1,7 @@
 
-OPT_DEFS += -DDICHOTEMY_PROMICRO
-DICHOTEMY_UPLOAD_COMMAND = while [ ! -r $(USB) ]; do sleep 1; done; \
+OPT_DEFS += -DDICHOTOMY_PROMICRO
+OPT_DEFS += -DCATERINA_BOOTLOADER
```

Comment:
This isn't actually needed anymore. 

This setting is now `BOOTLOADER_CATERINA`, and is set when you set the `BOOTLOADER` value. 

However, feel free to keep the other. 

```suggestion
```